### ISSUE TITLE
Network errors on tile fetch? sfos 4.3

### ISSUE THREAD

**Issue by poetaster** [sthocs/harbour-bikeme#3]
The errors below are repeated a number of times (...). seems to be tile data is not loading? Visually, we see the bikes on the map, but not the map itself.

Works on 3.4

[W] unknown:0 - QNetworkReplyImplPrivate::error: Internal problem, this method must only be called once.
...
[W] unknown:0 - QHttpNetworkConnectionPrivate::_q_hostLookupFinished could not dequeu request
...
[W] unknown:0 - QGeoTileRequestManager: Failed to fetch tile (8802,5373,14) 5 times, giving up. Last error message was: 'Host otile1.mqcdn.com not found'
...

**Comment by sthocs** [comment]
Yeah, Jolla broke the `osm` plugin with Sailfish 4 when QtLocation was upgraded to 5.4 :slightly_frowning_face: 
https://forum.sailfishos.org/t/using-qml-map-osm-not-working/7261

I haven't removed the option in the hope that it would be fixed quickly, but it's almost been a year.
The workaround is to stick with HERE while it still works :grimacing:

**Comment by poetaster** [comment]
i'm going to move that forum article to a bug report. thanks!

**Comment by poetaster** [comment]
I tried : 
```
Map {
            Plugin {
                name: "osm"
                PluginParameter {
                    name: "osm.mapping.custom.host"
                    value: "https://tile.openstreetmap.org"
                }
            }
            activeMapType: supportedMapTypes[supportedMapTypes.length - 1]
```
But this does not work.

**Comment by poetaster** [comment]
Also tried this:
```

            Plugin {
                id : mapPlugin;
                name: "osm"
                PluginParameter { name: "osm.mapping.host"; value: "https://a.tile.openstreetmap.org" }

                parameters: [
                    PluginParameter { name: "app_id"; value: "" },
                    PluginParameter { name: "app_code"; value: "" }
                ]
            }
```
Which produces other interesting errors:

[W] unknown:0 - QObject::connect: Cannot connect (null)::copyrightsChanged(QImage,QPoint) to QDeclarativeGeoMapCopyrightNotice::copyrightsChanged(QImage,QPoint)

**Comment by sthocs** [comment]
I had forgotten about this issue, but as you had mentioned on the forum, it's been fixed with Sailfish 4.5 (thanks to you if I followed correctly 🙂). Closing!
